packages/client/e2e/mpe/test-a.spec.ts

```diff
@@ -37,82 +47,196 @@ const knownSearches: KnownSearchesRecord = {
             duration: 0,
         },
     ],
+
+    'BB Brunes': [
+        {
+            id: 'X3VNRVo7irM',
+            title: 'BB BRUNES - Dis-Moi [Clip Officiel]',
+            artistName: 'BBBrunesMusic',
+            duration: 0,
+        },
+        {
+            id: 'mF5etHMRMMM',
+            title: 'BB BRUNES - Coups et Blessures [Clip Officiel]',
+            artistName: 'BBBrunesMusic',
+            duration: 0,
+        },
+        {
+            id: '1d3etBBSSfw',
+            title: 'BB BRUNES - Lalalove You [Clip Officiel]',
+            artistName: 'BBBrunesMusic',
+            duration: 0,
+        },
+        {
+            id: 'DyRDeEWhW6M',
+            title: 'BB BRUNES - Aficionado [Clip Officiel]',
+            artistName: 'BBBrunesMusic',
+            duration: 0,
+        },
+        {
+            id: 'Qs-ucIS2B-0',
+            title: 'BB BRUNES - Stéréo [Clip Officiel]',
+            artistName: 'BBBrunesMusic',
+            duration: 0,
+        },
+    ],
 };
 
 async function createMpeRoom({
```

Comment:
Why is this using it's own createMpeRoom method ? Shouldn't it be using the mpe-test-utils.ts one ?